src/main/resources/templates/fragments/entry-table.html

```diff
@@ -4,6 +4,10 @@
 <body>
 <div th:fragment="entry-table(content)">
     <table class="entry">
+        <colgroup>
+            <col class="field-column" />
```

Comment:
sorry, this is nonsense. the `/` will appear in the output but doesn't seem to cause problems.